pylintrc

```diff
@@ -14,9 +14,6 @@ reports=no
 # too-many-arguments:
 #       Due to the nature of the CLI many commands have large arguments set which reflect in large arguments set in corresponding methods.
 #
-# too-many-lines:
```

Comment:
Could you explain why you remove this one?

Reply:
It's comment I added in previous move [[CI] Move 1: Copy job SourceStaticPy from Travis to ADO (1/6)](https://github.com/Azure/azure-cli-extensions/pull/1133).

Not take effect now.  And sepcifically for **_src/aks-preview/azext_aks_preview/custom.py_**.
I found out it's unnecessary for this file. cause found the solution for disable too-many-lines suggestion on pylint==1.9.5